### ISSUE TITLE
[Enhancement Request] Adding notes to tasks.

### ISSUE THREAD

**Issue by pawel-kraw** [scylladb/scylla-manager#3645]
Right now if someone updates a task, there's no easy way to track who performed the change and why some task was updated/disabled etc. Having an option to add a note when updating/creating tasks would be very convenient as it'd be clear why certain task was modified. For example:
```
sctool repair update -c <cluster_name> --enabled=false --note="disabling to test something" --author="name surname" <repair/task_id>
```

**Comment by mykaul** [comment]
@tzach - sounds like a basic auditing feature to me actually.

**Comment by tzach** [comment]
Yes. Its make sense to write such operations to log, similar to Scylla Enterprise audit to syslog option.
No need for an audit table.

**Comment by Michal-Leszczynski** [comment]
Fixed by #3934.
